cmd/tracee-ebpf/main.go

```diff
@@ -87,12 +88,11 @@ func main() {
 			cfg.Output = &output
 
 			// environment capabilities
-
-			selfCap, err := getSelfCapabilities()
+			selfCap, err := capabilities.Self()
 			if err != nil {
 				return err
 			}
-			if err = checkRequiredCapabilities(selfCap); err != nil {
```

Comment:
I think we can also remove `checkRequiredCapabilities` and `getSelfCapabilities` methods from main.go.